tests/installation/setup_online_repos.pm

```diff
@@ -55,7 +61,10 @@ sub run() {
 
     send_key $cmd{next};    # Next
 
-    assert_screen "desktop-selection", 200;    # Make sure repos setup is finished and went to next step already
+    if (get_var("WITH_MAIN_REPO")) {
```

Comment:
it's me! I don't remember why I use those generic name WITH_XXX_REPO, maybe I have not better idea at that moment.